resources/js/admin/settings/components/SettingsRange.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="setting-text">
+        <div v-if="input === null || !input.length" class="font-italic text-black-50">
```

Comment:
this could be simplified by 
```suggestion
        <div v-if="!!input" class="font-italic text-black-50">
```

Reply:
NICE!!! 
Thank you!